### PR TITLE
Update examples for 0.2.

### PR DESCRIPTION
**Pull request by jlewi**
* Make tf-job-simple a v1alpha2 prototype
* Rename the existing tf-job-simple-v1alpha1 prototype
* Add a prototype for tensorboard.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1035)

### CODE REVIEW

File: kubeflow/examples/prototypes/tensorboard.jsonnet [hunk 1]
```diff
@@ -0,0 +1,87 @@
+// @apiVersion 0.1
+// @name io.ksonnet.pkg.tensorboard
+// @description Prototype for Tensorboard deployments
+// @shortDescription Prototype for Tensorboard deployments
+// @param name string Name to give to the tensorboard deployment
+// @optionalParam logDir string /you/need/to/set/logDir  The path to use
```
Comment (ankushagarwal) [inline kubeflow/examples/prototypes/tensorboard.jsonnet]: logDir should probably be a mandatory param

**Comment by jlewi** [comment]
Thanks @ankushagarwal PTAL

**Comment by ankushagarwal** [comment]
[This](https://github.com/kubeflow/kubeflow/blob/master/testing/tf_job_simple_test.py#L62) should be changed to `util.run(["ks", "generate", "tf-job-simple-v1alpha1", "tf-job-simple"])` for the tests to pass

**Pull request merged**